src/test/java/picard/analysis/CollectMultipleMetricsTest.java

```diff
@@ -9,27 +9,33 @@
 import htsjdk.samtools.SAMTextHeaderCodec;
 import htsjdk.samtools.metrics.MetricsFile;
 import htsjdk.samtools.util.BufferedLineReader;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.IntervalList;
+import htsjdk.samtools.util.StringUtil;
+import org.apache.commons.lang3.StringUtils;
 import org.testng.Assert;
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
+import picard.annotation.RefFlatReader;
```

Comment:
I don't see where this is used. Probably want to "optimize imports" or whatever.